packages/ibm-products/src/components/EditTearsheet/EditTearsheetForm.tsx

```diff
@@ -37,10 +84,10 @@ export let EditTearsheetForm = forwardRef(
 
       // Collect any other property values passed in.
       ...rest
-    },
-    ref
+    }: EditTearsheetFormProps,
+    ref: ForwardedRef<HTMLDivElement>
   ) => {
-    const formContext = useContext(FormContext);
+    const formContext = useContext(FormContext) as any;
```

Comment:
You could avoid using `any` here if you provide a type that is shared between when this context is created in `EditTearsheet` and then here as well.
```
export const FormContext = createContext<FormContextType | null>(null);
```

```
const formContext = useContext(FormContext) as FormContextType;
```

where `FormContextType` could be something like
```
type FormContextType = {
  currentForm: number;
  setFormTitle: () => void;
};
```